examples/hmrc_manual_section/frontend/vatgpb2000.json

```diff
@@ -1,55 +1,53 @@
 {
   "content_id": "2189bae1-a94e-4f90-917b-ffe25ec9ac4d",
-  "base_path": "/hmrc-internal-manuals/vat-government-and-public-bodies/vatgpb1000",
+  "base_path": "/hmrc-internal-manuals/vat-government-and-public-bodies/vatgpb2000",
```

Comment:
looks fine to me - but out of interest why does changing this to `2000` make it easier for testing?

Reply:
Thanks for the review Ed and good question. It makes it easier as the section groups are ordered and can be navigated through with previous and next page links (example page: https://www.gov.uk/hmrc-internal-manuals/vat-government-and-public-bodies/vatgpb2000). As this was the first section (vatgpb1000), only the next link was generated from the tests. I could've altered the data directly but thought it would be cleaner to just update the schema example.